package.json

```diff
@@ -66,7 +66,7 @@
         "mongoose": "^5.9.11",
         "mysql": "^2.18.1",
         "node-cron": "^2.0.3",
-        "node-mailer": "^6.4.6",
+        "nodemailer": "^6.4.6",
```

Comment:
Can you please check the version of nodemailer. Seems like 6.4.17 is the latest version. [Nodemailer documentation](https://www.npmjs.com/package/nodemailer)

Reply:
sure